config/geography/north_america.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package geography
+
+const (
+	// NorthAmerica is the HCP deployment in North America
+	NorthAmerica = "north_america"
```

Comment:
Shall we rename this file to `us.go` according to this [document](https://docs.google.com/document/d/1_Oj5RNLaN5mssmCSmfSwBR1j8E03uSFjmFwYCCg3U1Q/edit?tab=t.0#bookmark=id.nswoywq838bu)